solidity/interfaces/IStatefulChainlinkOracle.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+import '@chainlink/contracts/src/v0.8/interfaces/FeedRegistryInterface.sol';
+
+/// @title The interface for an oracle that provides price quotes
+/// @notice These methods allow users to add support for pairs, and then ask for quotes
+interface IPriceOracle {
```

Comment:
My only suggestions it would be to move this `///` comments to new `/**` format we are starting to use across-repos.

Reply:
Have another task for just that, I wanted this copy to be as similar as possible to the original